src/test/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitorTest.java

```diff
@@ -35,17 +37,23 @@
     private static final String TEST_PASSPHRASE = "testPassphrase";
     private static final String SUBJECT_PRINCIPAL
             = "CN=testCNC\\=USST\\=WashingtonL\\=SeattleO\\=Amazon.com Inc.OU\\=Amazon Web Services";
-    private static final long TEST_CERT_EXPIRY_CHECK_SECONDS = 1;
+    private static final long TEST_CERT_EXPIRY_CHECK_MILLIS = 200;
 
     @Mock
     private Consumer<X509Certificate> mockCallback;
 
     @Mock
     private ConnectivityInfoProvider mockConnectivityInfoProvider;
+    private final ScheduledExecutorService ses = new ScheduledThreadPoolExecutor(1);
 
     @TempDir
     Path tmpPath;
 
+    @AfterEach
+    void afterEach() {
```

Comment:
If its expected to be there for EACH test, it needs to be initialized again. Its can be confusing for future devs.

Reply:
No it doesn't need to be reinitialized.